source/reference/mongosync.txt

```diff
@@ -120,6 +120,15 @@ Global Options
 
       .. include:: /includes/fact-log-rotation-usr1-signal
 
+.. option:: --migrationName <name>
```

Comment:
**Suggestion**

I know this isnt part of your edit but I notice that the options specified on the page `/reference/mongosync/`  are a mix of cases. 

For example, `--id <ID>` uppercase and `--migrationName <name>` lowercase.

1. Could you correct in this PR  
2. Or file a separate ticket to correct in future?

Reply:
addressing that here!